VideoAPI/VideoApi.DAL/Mappings/RoomMap.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using VideoApi.Domain;
+
+namespace VideoApi.DAL.Mappings
+{
+    public class RoomMap : IEntityTypeConfiguration<Room>
+    {
+        public void Configure(EntityTypeBuilder<Room> builder)
+        {
+            builder.ToTable(nameof(Room));
+
+            builder.HasKey(x => x.Id);
+
+            builder.HasMany<RoomParticipant>("RoomParticipants").WithOne().OnDelete(DeleteBehavior.Cascade);
```

Comment:
Since `RoomParticipants` is public, it would be safer to user write the above as `builder.HasMany(x => x.RoomParticipants).WithOne().OnDelete(DeleteBehavior.Cascade);` rather than use strings

Reply:
done